lua/pulsar_lib/core/sql/sv_sql.lua

```diff
@@ -56,6 +56,38 @@ function SQL:Connect()
 	end
 end
 
+-- Manually replace ? with values in a query
+function SQL:prepareStatement(query, values)
+	local values = values or {}
+	local newQuery = ""
+	local i = 1
+	local last = 0
+
+	while true do
+		local start, stop = string.find(query, "?", last + 1, true)
+		if not start then break end
+		newQuery = newQuery .. string.sub(query, last + 1, start - 1)
+		local value = values[i]
+
+		if value == nil then
+			newQuery = newQuery .. "NULL"
+		elseif type(value) == "string" then
+			newQuery = newQuery .. self:Escape(value)
```

Comment:
Not sure if self:Escape wraps in quotes? Could this be checked please.

Reply:
No it doesnt, i fixed SQL:Escape to do the same in either sqlite or mysql settings

Reply:
If it's not wrapped, it'll just chuck a string without quotes into the query then? Probably needs a fix for that.